components/list/list-item-expand-collapse-mixin.js

```diff
@@ -33,10 +33,11 @@ export const ListItemExpandCollapseMixin = superclass => class extends SkeletonM
 				--d2l-expand-collapse-slot-transition-duration: 0.3s;
 			}
 			.d2l-list-expand-collapse {
+				line-height: 0;
 				width: 0;
 			}
 			:host([dir="rtl"][_render-expand-collapse-slot]) .d2l-list-expand-collapse {
-				padding: 0.4rem 0 0 0.3rem;
+				padding: 0.55rem 0 0 0.3rem;
```

Comment:
Really excited to start converting this stuff to logical properties...

Reply:
I almost did it but decided not to scope creep.